test/integration-test.sh

```diff
@@ -13,15 +13,12 @@ case $1 in
     # sadly, bootstrap.sh from release-1.4 still pulls binaries from Nexus, which is not available anymore
     # Why comment following code? Please check this issue: https://github.com/Hyperledger-TWGC/tape/issues/159
     # curl -vsS https://raw.githubusercontent.com/hyperledger/fabric/release-2.2/scripts/bootstrap.sh | bash
-    ./test/bootstraps/bootstrap-v2.2.sh
+    ./test/bootstrap-v2.2.sh 1.4.12 1.5.2
     cd ./fabric-samples/
     git checkout release-1.4
     cd ./first-network
-    # 1.4.10
-    echo y | ./byfn.sh up -i 1.4.10
-    # comments here for 1.4.8 work around as docker image issue.
-    # docker pull hyperledger/fabric-orderer:amd64-1.4  
-    # Error response from daemon: manifest for hyperledger/fabric-orderer:amd64-1.4 not found: manifest unknown: manifest unknown
+
+    echo y | ./byfn.sh up -i 1.4.12
```

Comment:
I suggest change back to `1.4.10`
is 1.4 still LTS?
are we going to upgrade CI changes with this pr?

Reply:
1.4 is no more LTS, would you consider a removal for this?

Reply:
So far not, as it is previous TLS, and I make a PR for supporting at https://github.com/Hyperledger-TWGC/tape/pull/250
but honestly speaking, for my point of view, as 1.4 is no previous LTS, we don't need to bump up version in CI in high frequency.

Reply:
1.4.12 should be last version of release-1.4, I think no further bump up in future.